package.json

```
@@ -62,22 +62,26 @@
     "prettier": "^2.2.1"
   },
   "scripts": {
-    "build": "next build",
+    "build": "yarn remove:commons && next build",
     "dev": "yarn dev:next",
     "dev:next": "next dev",
+    "submodule": "git submodule init && git submodule update",
     "export": "yarn run build && next export",
     "format": "yarn prettier --write --ignore-path './.gitignore' \"**/*.+(json|md|yml)\" './'",
     "format-staged": "prettier --write",
     "lint": "yarn eslint --fix --ignore-path './.gitignore' './'",
     "lint-staged": "yarn eslint --fix --ext '.js,.json,.md,.yml'",
-    "start": "next start"
+    "start": "next start",
+    "commons-setup": "node ./scripts/commons-ui-add.js && yarn && git restore package.json && git restore yarn.lock",
```

```suggestion
    "setup:commons": "node ./scripts/commons-ui-add.js && yarn && git restore package.json && git restore yarn.lock",
```
Or `"add:commons"`, etc., i.e. it should be a verb.